src/wp-includes/block-template-utils.php

```diff
@@ -252,7 +252,7 @@ static function ( $item ) {
 		$type,
 		WP_TEMPLATE_PART_AREA_UNCATEGORIZED
 	);
-	trigger_error( $warning_message, E_USER_NOTICE );
+	wp_trigger_error( __FUNCTION__, $warning_message, E_USER_NOTICE );
```

Comment:
`E_USER_NOTICE` is not longer needed as it's the default.
```suggestion
	wp_trigger_error( __FUNCTION__, $warning_message );
```